src/components/cars/CarDetails.jsx

```diff
@@ -0,0 +1,75 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { useNavigate, useParams } from 'react-router-dom';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faArrowCircleRight } from '@fortawesome/free-solid-svg-icons';
+import Navbar from '../Navbar';
+import SkeletonLoading from './SkeletonLoading';
+import { fetchCarById } from '../../redux/features/carsAction';
+import { reserveCar } from '../../redux/features/carsSlice';
+
+function CarDetails() {
```

Comment:
- [ ]  Please, kindly refactor this functional component to ES6 arrow function, so your code complies with the latest standards.

Refactoring is easy:

```javascript
const CarDetails= () => {
  // ...
}
```

## Explanation

The `function` keyword has been soft deprecated in almost all the software industry in favor of ES6 arrow functions. This means, almost nobody uses it anymore, so much that it's considered a non-best practice.

That way, your code will meet the latest standards. 💪